config/README.md

```diff
@@ -0,0 +1,140 @@
+## Veraison Services Configuration
```

Comment:
> (the indentation in the go snippets is gone a bit wild though.)

Erm, could you clarify? Both in source, and as rendered by GitHub, the  indentation seems to be ok to me?

Reply:
mmm, you are right that when seen through the github rendered it looks ok.  On my markdown thingy it looks a bit wonky possibly because there's a mix of tabs and spaces.
Don't worry.